src/integration/HubspotApiModel.ts

```diff
@@ -1,31 +1,44 @@
 import axios from "axios";
 import querystring from 'querystring';
 
+
 export class HubspotApiModel {
 
     
     // Temp access until DB has OAuth access tokens
-    private HAPIKEY: string;
+    // private HAPIKEY: string;
+    //
+    // constructor(apiKey: string){
+    //     this.HAPIKEY = apiKey;
+    // }
```

Comment:
Would it be possible to remove this comment? I don't think we should rely on api keys once we have OAuth setup.

Reply:
Sorry, I'm not sure why that is still showing up do you see the "Outdated" symbol? Perhaps you were looking at an older commit?

Reply:
I missed the outdated symbol you're right.